### PR TITLE
Add icon and desciption

### PR DESCRIPTION
**Pull request by JDOG787**
Hi, I saw the extension didnt have an icon so I added one. This should work because I looked it up, but I've never made an extension before, and theres not a good way to test to see if the icon works afaik. I tried making a logo, but feel free to change later. Let me know if theres any issues,

Thanks

### CODE REVIEW

**Comment by whippingdot** [comment]
Lol the icon looks sooo funny but nice at the same time

**Comment by JDOG787** [comment]
Haha thanks. I’m not great at making logos lol

**Comment by whippingdot** [comment]
Hey you wanna join the github org for us replers(we starting a new proj rn)

**Comment by JDOG787** [comment]
@masad-frost is this ok?

**Comment by whippingdot** [comment]
They don't check github often. you will probably get a reply by the end of this week

**Comment by masad-frost** [comment]
We check do check GitHub often, but there's priorities!

File: package.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "name": "replit",
   "displayName": "replit",
-  "description": "",
+  "description": "VS Code extension for connecting to repls using the crosis api",
```
Comment (masad-frost) [inline package.json:4]:
```suggestion
  "description": "VS Code extension for connecting to repls",
```

Should we mention crosis? Seems like an implementation detail that the user doesn't need to know about. Then again you need an API key to use this.

Comment (JDOG787) [reply package.json:4]: Hmmm, true. Might be better not to say, but up to you.

Comment (JDOG787) [reply package.json:4]: I'll go ahead, and get rid of it, but we can change it later

**Comment by JDOG787** [comment]
@masad-frost Ok done!

**Review by masad-frost — APPROVED** [review]
Looks good. Thank you.

I'll ship it with the next update!

**Comment by masad-frost** [comment]
I just saw the logo. I thought it was just the replit logo, looks cool, thanks 😊

**Comment by whippingdot** [comment]
Sorry but I thought you guys don't check GitHub often as you have to work on repl.it. I didn't want that to come out mean, you have to work at repl.it and so you might not have much time to respond.

**Comment by JDOG787** [comment]
Awesome thanks!

**Comment by masad-frost** [comment]
@whippingdot not at all! That came out as considerate

**Pull request merged**